packages/@o3r/localization/src/tools/localized-currency.pipe.spec.ts

```diff
@@ -31,6 +31,25 @@ import {
   LocalizedCurrencyPipe
 } from './localized-currency.pipe';
 
+/**
+ * Fixture for ChangeDetectorRef
+ */
+class ChangeDetectorRefFixture implements Readonly<ChangeDetectorRef> {
```

Comment:
Fallback changes removed in previous PR because `jest/unbound-method` was not activated at this time

Reply:
sounds like a reusable class. could you create a task to factorize it ?

Reply:
That's for testing we use it three times but we don't have to mock like that every time
I just don't want to refactor the test